models/schemas/Bundle.yaml

```diff
@@ -47,6 +47,7 @@ properties:
           example: "urn:uuid:CF464979-41A7-4231-B70F-AAE377A973C9"
         resource:
           anyOf:
+            - $ref: "MessageHeader.yaml"
```

Comment:
Because we're only dealing with bundles of type "message", should the rest of this file be updated to reflect that?